src/services/playbookService.js

```diff
@@ -29,7 +29,7 @@ export const GetComparisonValue = (condition) => {
       return condition.comparisonValue
     case 1: // datetime
       return condition.dateTimeComparisonValue
-                ? moment.utc(condition.dateTimeComparisonValue)
+                ? DateTime.fromISO(condition.dateTimeComparisonValue)
```

Comment:
I'm going to preface this with: I'm sorry for asking you to fix my bug.

This function is trying to get an object to use in a comparison in testableTestByConditionType below.

Unfortunately, testableTestByConditionType will grab a string value as condition.value even when the condition is attempting to compare datetimes. Is checking comparisonValue for being a DateTime instance and calling DateTime.fromISO(condition.value) to make const value a datetime instance and then doing whatever approved Luxon dateTime comparison rather than the standard JavaScript (value > comparisonValue) in scope for this pull request, or should we open a separate bug for it?

Reply:
I don't think I understand this bug, and it was not introduced by this change. Can you open  followon ticket with reproduction steps?